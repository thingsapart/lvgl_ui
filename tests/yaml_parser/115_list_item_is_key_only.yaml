- key:
- another_key: value
