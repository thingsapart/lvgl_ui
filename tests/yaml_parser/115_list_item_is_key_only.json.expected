[{
		"key":	null
	}, {
		"another_key":	"value"
	}]
